catalog/spatial/registry/registry-schema-bindings/pom.xml

```diff
@@ -154,29 +184,33 @@
                         </goals>
                         <configuration>
                             <haltOnFailure>true</haltOnFailure>
+                            <includes>
```

Comment:
Do we really need to use `<includes>` here? If so, can we use a path instead of specific file names? That way if someone adds a new class down the road, it will be included by default.